### DISCUSSION TITLE
Welcome to Discussions!

### DISCUSSION THREAD

**Discussion by phanxuanquang** [phanxuanquang/Adobe-Camera-Profiles-Unlocker#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by inickme** [comment]
Bạn ơi support Nikon Zf giúp mình với, hiện tại chỉ có Zfc, thanks bạn

**Reply by phanxuanquang** [reply]
Bạn tải lại bản mới nhất thử xem

**Comment by THICH-AI** [comment]
Mình cài thử bản mới nhất trên window 10, chạy thử SUCCES nhưng sau đó không thấy gì cả trong C:\Users\xxx\AppData\Roaming\Adobe\CameraRaw và C:\Users\xxx\AppData\Roaming\Adobe\Lightroom

**Reply by phanxuanquang** [reply]
Bạn xem trong Camera Raw/Lightroom đã có chưa

**Reply by THICH-AI** [reply]
Cũng không thấy gì cả, không hiểu file profiles tạo ra có bị copy nhầm sang thư mục khác không. Mình dùng lightroom với photoshop version mới nhất

**Reply by THICH-AI** [reply]
mình xem trên page này thấy có vài bạn bảo chạy xong cũng không thấy gì cả: https://www.facebook.com/NguyenHoangPhucPhoto/videos/1255169492252628

**Reply by phanxuanquang** [reply]
Bạn thử cài lại Camera Raw bản mới nhất, và Photoshop/Lightroom bản 2022 về sau nhé

**Reply by THICH-AI** [reply]
mình đang dùng camera raw bản 17, pts 26 và Lr 14, là những bản mới nhất đó

**Reply by phanxuanquang** [reply]
Nếu bạn chọn profile của Fujifilm, Nikon, Panasonic, Adobe, Sigma thì nó sẽ không được lưu ở những folder đó nha

**Reply by quangmint** [reply]
Mình thử thì thấy profile màu của mọi dòng máy đều tạo được trừ của Fuji, cứ tạo của fuji là không thấy file nào trong ổ hết. Bạn check thử xem sao nhé.
